### PR TITLE
[editor] Make single point amenity=parking addable

### PR DESCRIPTION
**Pull request by pastk**

### CODE REVIEW

**Comment by vng** [comment]
I have mixed feelings here :)

**Comment by pastk** [comment]
> I have mixed feelings here :)

Please check a discussion in https://github.com/organicmaps/organicmaps/issues/2781

File: data/editor.config [hunk 1]
```diff
@@ -287,8 +287,9 @@
         <include field="operator" />
         <include field="postcode" />
       </type>
-      <type id="amenity-parking" can_add="no">
+      <type id="amenity-parking">
         <include field="name" />
+        <include field="operator" />
```
Comment (biodranik) [inline data/editor.config:292]: Opening hours?

Comment (biodranik) [reply data/editor.config:292]: Access? Fee?

**Review by biodranik — APPROVED** [review]
Usually single-car parking spaces are rare, so using a point maybe not be the best option. But on an empty map, it's better to have parking as a point than nothing...

**Comment by pastk** [comment]
Added opening_hours and made bicycle and motorcycle parkings use the same fields.

File: data/editor.config (hunk 1 above)
Comment (pastk) [reply data/editor.config:292]: We don't have access and fee fields support in the editor yet.

**Review by biodranik — APPROVED** [review]

**Pull request merged**